bin/templates/cordova/lib/pluginHandlers.js

```diff
@@ -320,26 +320,33 @@ function generateAttributeError (attribute, element, id) {
 function getInstallDestination (obj) {
     var APP_MAIN_PREFIX = 'app/src/main';
 
-    if (obj.targetDir.startsWith('app')) {
+    var targetDirArray = obj.targetDir.split('/');
```

Comment:
I would favor it if you would define '/' as a var or constant above, like I defined `APP_MAIN_PREFIX`.

Reply:
P.S. It would be nice if we could have used the constant in the regexs below, but I think it is not possible. Maybe no need to define the constant then.

Reply:
The "new RegExp" should probably be its own variable as well, to make it more readable.

Reply:
And I would not favor backticks since they would not work on deprecated Node.js 4. (We cannot drop Node.js 4 from 7.1.x release. Even if you rebase it on master, I would rather keep it without backticks so we can cherry-pick it to 7.1.x if needed.)